app/editor/src/features/admin/event-of-the-day/hooks/useColumns.tsx

```diff
@@ -0,0 +1,195 @@
+import { calcTopicScore } from 'features/content/form/utils';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useLookup } from 'store/hooks';
+import {
+  CellDate,
+  FieldSize,
+  getSortableOptions,
+  IContentTopicModel,
+  IFolderContentModel,
+  ITableHookColumn,
+  OptionItem,
+  Select,
+} from 'tno-core';
+
+export const useColumns = (
+  handleSubmit: (values: IFolderContentModel) => Promise<void>,
+  loading: boolean,
+): ITableHookColumn<IFolderContentModel>[] => {
+  const [{ topics, rules }] = useLookup();
+  const [folderContentModel, setFolderContentModel] = React.useState<IFolderContentModel>();
+
+  // create an array with values 0-100 - min score is 0, max is 100
+  const possibleScores = Array.from(Array(101).keys()).map(
```

Comment:
I'd move this out so that it's a constant or a function.  Right now it'll re-render each time.